docs/pages/table-data.rst

```diff
@@ -74,6 +74,15 @@ what fields to show or hide:
 - `~.Table.Meta.fields` -- specify model fields to *include*
 - `~.Table.Meta.exclude` -- specify model fields to *exclude*
 
+In this example we will demonstrate how this can be done::
+    
+    # tables.py
+    class PersonTable(tables.Table):
+    class Meta:
+        model = Person
+        sequence = ("last_name", "first_name", "birth_date", )
+        exclude = ("user", )
```

Comment:
Besides the indentation issue for which I suggest a change below, I think we need to describe what the user should expect from this example. Just adding an example does not clarify the interaction of `sequence`, `fields` and `exclude` much.

Can you add such an explanation?

```suggestion
    class PersonTable(tables.Table):
        class Meta:
            model = Person
            sequence = ("last_name", "first_name", "birth_date", )
            exclude = ("user", )
```

Reply:
Oops, thanks for catching this. I've accepted the suggestion and will now add an explanation.

Reply:
@jieter Explanation added - let me know if that's what you had in mind. I'm happy to revise it further if needed.